app/models/answer_set.rb

```diff
@@ -4,12 +4,12 @@ class AnswerSet
   attr_accessor :questioning, :answers
 
   delegate :qtype, :required?, :question, :condition, to: :questioning
-    delegate :name, :hint, to: :question, prefix: true
-    delegate :option_set, to: :question
-      delegate :levels, to: :option_set
+  delegate :name, :hint, to: :question, prefix: true
+  delegate :option_set, to: :question
+  delegate :levels, to: :option_set
   delegate :first, to: :answers
-    delegate :errors, :choices, :all_choices, :value, :datetime_value, :date_value, :time_value, :response_id, :questioning_id, :relevant, to: :first
-    delegate :option_ids_with_no_nils, to: :option_path
+  delegate :errors, :choices, :all_choices, :value, :datetime_value, :date_value, :time_value, :response_id, :questioning_id, :relevant, to: :first
+  delegate :option_ids_with_no_nils, to: :option_path
 
   # Builds Answer attribute hashes from submitted answer_set params.
   # Returns an array of Answer attribute hashes.
```

Comment:
This model is a shitshow. Would love to brainstorm on how to refactor it, someday.